src/style.css

```diff
@@ -1 +1,65 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document